site/guides/porch-ui-installation.md

```diff
@@ -0,0 +1,52 @@
+# Accessing the Configuration as Data UI
```

Comment:
I am wondering if we should refer the UI as `Package Orchestrator UI` ?

Reply:
@bgrant0607 Thoughts?

Reply:
The `Configuration as Data` name I have already started publishing with, so unless we will be only referring to the UI as the `Package Orchestrator` within kpt.dev, renaming is starting to get to the point where it becomes difficult (although not impossible).

Reply:
"Package Orchestrator" is not going to be a thing we advertise to users. It's a component. It's relevant to contributors. Let's stay with "Configuration as Data" for now. That's the overall approach, vs IaC. And "kpt" is the overall project name, so that would be the other option.

Reply:
I know we have the user guide discussing porch for now, that's fine.

Reply:
Ack.